lib/utils.js

```diff
@@ -36,20 +56,30 @@ function getChangelogSectionIndex (lines) {
 }
 
 /**
+ * Get the given key from process.env, substituting "undefined" for the real undefined
+ * @param {String} key - the environment variable we want to get
+ * @returns {*} whatever is at process.env[key] with the one exception of "undefined" being translated to undefined
+ */
+function getEnv (key) {
+  const value = process.env[key]
+  return (value === 'undefined') ? undefined : value
```

Comment:
I think they did, but in some cases we got an "undefined" string when really we wanted `undefined`. In this case, an undefined/null/0 will still get through, but a string "undefined" will also be considered `undefined`. I noticed that in some cases, the `process.env` ended up with "undefined" values when I didn't think they should be set at all.